src/components/Calculator.js

```diff
@@ -1,69 +1,57 @@
-import { Component } from 'react';
+import { useState } from 'react';
 import calculate from '../logic/calculate';
 import Display from './display';
 import Button from './button';
 // eslint-disable-next-line react/prefer-stateless-function
-export default class Calculator extends Component {
-  constructor(props) {
-    super(props);
-    this.state = {
-      total: null,
-      next: null,
-      operation: null,
-    };
-    this.buttonClick = this.buttonClick.bind(this);
-  }
-
-  componentDidMount() {
-    this.setState({ total: '0' });
-  }
+const Calculator = () => {
```

Comment:
The Calculator component (components/Calculator.js) is refactored from a class-based component to a functional component using [hooks](https://reactjs.org/docs/hooks-intro.html) ✔